static/js/js-old/js4.js

```diff
@@ -426,7 +426,7 @@ function buildOptions($cont, $slides, els, options, o) {
 			$cont.css({height:maxh+'px'});
 	}
 
-	var pauseFlag = false;  // https://github.com/malsup/cycle/issues/44
+	var pauseFlag = false;  // http://github.com/malsup/cycle/issues/44
```

Comment:
Please revert all changes you made as comments as those shouldn't matter and just clutter up this PR.